ontology/shacl_constraints.ttl

```diff
@@ -0,0 +1,61 @@
+@prefix ex: <http://datashapes.org/shasf/tests/expression/advanced.test.shacl#> .
+@prefix exOnt: <http://datashapes.org/shasf/tests/expression/advanced.test.ont#> .
+@prefix exData: <http://datashapes.org/shasf/tests/expression/advanced.test.data#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+<http://datashapes.org/shasf/tests/expression/advanced.test.shacl>
```

Comment:
Please move this to some prefix that makes sense for the OpenCS ontology. Check what are the other prefixes used in OpenCS.